### PR TITLE
table: add GeneratedExpr in table.Column.

### PR DESCRIPTION
**Pull request by hicqu**
Why don't add GeneratedExpr in `model.ColumnInfo`?
Because in module `model`, we can't import things from `ast`. So I add it into `table.Column`, just like we add other things into `table.Table`.

@shenli @coocood @hanfei1991 , PTAL, thanks.

### CODE REVIEW

File: table/tables/tables.go [hunk 1]
```diff
@@ -71,6 +72,11 @@ func TableFromMeta(alloc autoid.Allocator, tblInfo *model.TableInfo) (table.Tabl
 		}
 
 		col := table.ToColumn(colInfo)
+		if len(colInfo.GeneratedExprString) != 0 {
+			// here we can ignore error because GeneratedExprString has been already
+			// checked when this table or column was added.
+			col.GeneratedExpr, _ = parser.ParseExpression(colInfo.GeneratedExprString)
```
Comment (coocood) [inline table/tables/tables.go]: We should check the error, it not worth to spend time thinking if it is safe to not check the error.

File: table/tables/tables.go [hunk 2]
```diff
@@ -71,6 +72,14 @@ func TableFromMeta(alloc autoid.Allocator, tblInfo *model.TableInfo) (table.Tabl
 		}
 
 		col := table.ToColumn(colInfo)
+		if len(colInfo.GeneratedExprString) != 0 {
+			// here we can ignore error because GeneratedExprString has been already
+			// checked when this table or column was added.
+			var err error
+			if col.GeneratedExpr, _ = parser.ParseExpression(colInfo.GeneratedExprString); err != nil {
```
Comment (coocood) [inline table/tables/tables.go]: The error is not assigned.

Comment (coocood) [inline table/tables/tables.go]: Usually, we prefer not to put the execution line into if condition.
We can define `err1` to avoid name conflict.

Comment (hicqu) [inline table/tables/tables.go]: fixed. sorry for that.

**Comment by coocood** [comment]
LGTM

**Pull request merged**